openlibrary/templates/merge/authors.html

```diff
@@ -90,13 +90,19 @@ <h1>$_("Merge Authors")</h1>
                         <p>${ macros.TruncateString(a.bio, 250) }</p>
                     $else:
                         <p>No description.</p>
-                    <ul>
-                    $for doc in top.docs:
-                        <li><a href="$doc['key']" target="new" title="$_('Open in a new window')">$doc['title']</a> <span class="smaller">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
-                        $if doc.get('first_publish_year'):
-                            <span title="$_('First published in')">$doc['first_publish_year']</span>
-                        </span></li>
-                    </ul>
+                        <ul>
+                            $for doc in top.docs:
+                                $if is_rtl_lang:
+                                    <li><a href="$doc['key']" target="new" title="$_('Open in a new window')" dir="rtl">$doc['title']</a> <span class="smaller" dir="rtl">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
+                                        $if doc.get('first_publish_year'):
+                                            <span title="$_('First published in')" dir="rtl">$doc['first_publish_year']</span>
+                                        </span></li>
+                                $else:
+                                    <li><a href="$doc['key']" target="new" title="$_('Open in a new window')">$doc['title']</a> <span class="smaller">$ungettext('%(count)s edition', '%(count)s editions', doc['edition_count'], count=doc['edition_count']),
+                                    $if doc.get('first_publish_year'):
+                                        <span title="$_('First published in')">$doc['first_publish_year']</span>
+                                    </span></li>
+                        </ul>
```

Comment:
And even if it _was_ the proper way to do it (again, it’s not), this would be a lot of repeated code. There must be [a DRYer way](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to accomplish this.